core/src/main/java/org/apache/gravitino/storage/relational/service/MetadataObjectService.java

```diff
@@ -214,4 +233,407 @@ public static String getMetadataObjectFullName(String type, long metadataObjectI
 
     return fullName;
   }
+
+  /**
+   * Retrieves a map of Metalake object IDs to their full names.
+   *
+   * @param ids A list of Metalake object IDs to fetch names for.
+   * @return A Map where the key is the Metalake ID and the value is the Metalake full name. The map
+   *     may contain null values for the names if its parent object is deleted. Returns an empty map
+   *     if no Metalake objects are found for the given IDs. {@code @example} value of metalake full
+   *     name: "metalake1.catalog1.schema1.table1"
+   */
+  public static Map<Long, String> getMetalakeObjectsFullName(List<Long> ids) {
```

Comment:
better to change `List<Long> ids` -> `List<Long> metalakeIds`, the same below.

Reply:
Ok, all input parameter names in `getXXXObjectsFullName` is changed.